.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Fixes # 
```

Comment:
Can we remove this title? Pull Requests won't always be a fix. Sometimes it might be a feature.
```suggestion

```